.kubernetes/kubernetes_ingresses.yaml

```diff
@@ -35,6 +35,10 @@ spec:
           serviceName: fakebook-profiles-api
           servicePort: 80
         path: /api/profiles
+      - backend:
+          serviceName: fakebook-profile-picture-api
```

Comment:
the service is named `facebook-profiles-api`.

Reply:
Fixed.